src/components/Card/Card.tsx

```diff
@@ -41,19 +62,23 @@ export const Card = ({
   return (
     <section
       onClick={toggleSelected}
-      className={`${cardClass} ${selected ? `border-${tagColor}` : ''} `}>
+      className={`${cardClass} ${selected ? `card--${type}` : ''} `}>
       <figure className='card__container-image'>
-        <img className='card__image' src={imgSrc} alt={imgAlt} />
+        <div
+          className='card__image'
+          style={{ backgroundImage: `url(${src})` }}
+          role='img'
+          aria-label={alt}
+          title={alt}
+        />
       </figure>
```

Comment:
Aż sprawdziłem, czy `div` może być wewnątrz `figure` i jak najbardziej może. Fajnie, że zadbałeś o a11y. 👍 
> **Content model:**
> Either: one figcaption element followed by flow content.
> Or: flow content followed by one figcaption element.
> Or: flow content.
— https://html.spec.whatwg.org/multipage/grouping-content.html#the-figure-element